src/gui/folder.cpp

```diff
@@ -879,7 +879,7 @@ void Folder::correctPlaceholderFiles()
         return;
     }
     static const auto placeholdersCorrectedKey = QStringLiteral("placeholders_corrected");
-    const auto placeholdersCorrected = _journal.keyValueStoreGetBool(placeholdersCorrectedKey, false);
+    const auto placeholdersCorrected = _journal.keyValueStoreGetInt(placeholdersCorrectedKey, false);
```

Comment:
@matthieugallien should the default value really be `false`? The parameter is accepting `qint64`, shouldn't `false` be `0`?

Reply:
not sure if there is really a difference
I kept it because it was already there